atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -326,43 +332,93 @@ def get_buildroot(self, build_id):
 
         return buildroot
 
+    def parse_orchestrator_logs(self, logs):
+        """
+        Parse the output of get_orchestrator_build_logs()
+
+        :return: list, Output instances
+        """
+        output = []
+
+        # Build list of arches
+        arches = []
+        for log in logs:
+            if log.platform is not None and log.platform not in arches:
+                arches.append(log.platform)
```

Comment:
Could also write this as `arches = set([log.platform for log in logs if log.platform is not None])`